config.js

```diff
@@ -95,7 +95,8 @@ const ConfigOptions = () => {
             js: {},
             css: {},
             img: {},
-            fonts: {}
+            fonts: {},
+            docs: {}
```

Comment:
Can you add some unit tests for this new property, please?

Reply:
👍 Done